cpp/src/slice2cpp/Gen.cpp

```diff
@@ -2334,11 +2286,14 @@ Slice::Gen::ProxyVisitor::visitOperation(const OperationPtr& p)
     C << inParamsImplDecl << ("const " + getUnqualified("::Ice::Context&", interfaceScope) + " context");
     C << epar << " const";
     C << sb;
+    // TODO: switch to string_view and constexpr.
```

Comment:
Switching to string_view requires some work in OutgoingAsync. Maybe it could use a string_view too.